translate/storage/test_pypo.py

```diff
@@ -181,6 +181,17 @@ def test_wrap_on_max_line_length(self):
         print(str(unit))
         assert str(unit) == expected
 
+    def test_wrap_on_slash(self):
+        """test that we wrap on /"""
+        string = "1/3/5/7/N/" * 11
+        expected = 'msgid ""\n%s\nmsgstr ""\n' % '"1/3/5/7/N/1/3/5/7/N/1/3/5/7/N/1/3/5/7/N/1/3/5/7/N/1/3/5/7/N/1/3/5/7/N/1/3/5/"\n"7/N/1/3/5/7/N/1/3/5/7/N/1/3/5/7/N/"'
```

Comment:
Make the expected string be a triple quoted string, and avoid string interpolation.